src/main/java/au/com/dmg/fusion/response/paymentresponse/PaymentResponse.java

```diff
@@ -115,8 +116,8 @@ public Builder response(Response response) {
             return Builder.this;
         }
 
-        public Builder saleData(PaymentResponseSaleData saleData) {
-            this.saleData = saleData;
+        public Builder saleData(SaleData saleData) {
```

Comment:
Hey Grace, I cast it here so that it would not need to be explicitly casted when building paymentResponse. When I update I implement the change, the test classes are having errors and requiring me to explicitly cast the .saledata build to PaymentResponseSaleData.